src/services/measure/measurearea.js

```diff
@@ -0,0 +1,84 @@
+goog.provide('ngeo.interaction.MeasureArea');
+
+goog.require('ngeo.interaction.Measure');
+goog.require('ol.geom.Polygon');
+goog.require('ol.interaction.Draw');
+
+
+
+/**
+ * @classdesc
+ * Interaction dedicated to measure length.
+ *
+ * @constructor
+ * @extends {ngeo.interaction.Measure}
+ * @param {{
+ *    startMsg: (string|undefined),
+ *    continueMsg: (string|undefined),
+ *    style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    sketchStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)
+ * }=} opt_options Options
+ * @export
+ */
+ngeo.interaction.MeasureArea = function(opt_options) {
+
+  var options = goog.isDef(opt_options) ? opt_options : {};
+
+  goog.base(this, options);
+
+  this.drawInteraction = new ol.interaction.Draw(
+      /** @type {olx.interaction.DrawOptions} */ ({
+        type: 'Polygon',
+        features: this.overlay.getFeatures(),
+        style: options.sketchStyle
+      }));
```

Comment:
I think it would be cleaner to pass the draw interaction to the parent constructor, and have `drawInteraction` be a private member of the parent class.